.github/workflows/changelog_verification.yml

```diff
@@ -0,0 +1,40 @@
+name: Changelog Verification
+on:
+  pull_request:
+    branches:
+      - '**/release-**'
+  push:
+    branches:
+      - '**/release-**'
```

Comment:
pls get rid of the leading `**/`

release branches should start with `release-` with no personal prefix before it